crates/gossip/src/config.rs

```diff
@@ -71,6 +82,7 @@ impl Default for K2GossipConfig {
     fn default() -> Self {
         Self {
             max_gossip_op_bytes: 100 * 1024 * 1024,
+            max_request_gossip_op_bytes: 250 * 1024 * 1024,
```

Comment:
I would expect this to match the default of max bytes we can request from someone else, ergo 100. What do you think?

Reply:
Yeah I went back and forth on this a bit. My thinking is that you can get quite a few ops for 100Mb. But Holochain enforces a 1Mb limit on entry size. So you could get < 100 ops back with a 100Mb limit. I thought it maybe made sense to leave a little extra room for nodes that want to serve a DHT faster to its peer to return more data. 

Equally, this is a default... so maybe if a node wants to do that then it should just configure this limit up itself. Yeah okay, setting this to the same thing makes sense